jenkins-pipelines/operator/functional-eks.jenkins

```diff
@@ -0,0 +1,15 @@
+#!groovy
+
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'k8s-eks',
+    aws_region: 'eu-north-1',
+    test_name: 'scylla_operator',
+    test_config: 'test-cases/scylla-operator/functional.yaml',
+    email_recipients: 'qa@scylladb.com,scylla-operator@scylladb.com',
+    post_behavior_db_nodes: 'destroy',
+    post_behavior_loader_nodes: 'destroy',
+    post_behavior_monitor_nodes: 'destroy',
+    timeout: [time: 400, unit: 'MINUTES']
```

Comment:
timeout in case of longevity pipeline is calculated in another "vars" file. No need to provide it here.

Reply:
removed